Caltech-IPAC/firefly
# Firefly-548: improve click by click readout copying

### Firefly-548: improve click by click readout copying

  - added clipboard icon for lock by click
  - cleaned up cursor feedback
  - Target takes more flexible coordinate system naming
  - Additionally: Cleaned up MagnifiedView.jsx
  - Additionally: Cleaned up ThumbnailView.jsx

_ticket:_ https://jira.ipac.caltech.edu/browse/FIREFLY-320

### Testing
_url:_ https://fireflydev.ipac.caltech.edu/firefly-320-coord/firefly/

  - bring up any number of images
  - change mouse readout to `lock by click`
  - click on image
    - notice copy icon in the mouse readout
    - try the copy by clicking on the icon
    - try pasting in to the target panel
    - try it again with galactic
